redfish/rules/dell_redfish_1.17.0.yml

```diff
@@ -0,0 +1,431 @@
+Traverse:

```

Comment:
It seems that you have executed `collector` differently than I expected.
The generated rule should be sorted alphabetically.
Let's have a meeting to build a common understanding and to refine the how-to document.

Reply:
Thank you!
I am sorry, and not sure understand the document of corrector tools.  I am waiting you on Kubernetes Zoom meeting.